### ISSUE TITLE
Leftover status when test-only set

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit-service/packit-service#440]
- When a user removes `copr-build` job, packit still set the pending build status for the build targets.
- The test targets have the right state ("success").

Example PR: https://github.com/psss/tmt/pull/129
![Screenshot from 2020-02-27 12-58-05](https://user-images.githubusercontent.com/20214043/75443039-e57b0a80-5960-11ea-8d49-08967877484a.png)
![Screenshot from 2020-02-27 12-57-55](https://user-images.githubusercontent.com/20214043/75443041-e613a100-5960-11ea-984d-79c716add8f5.png)

**Comment by lachmanfrantisek** [comment]
It looks like it does not occur anymore.

Here is a list of flags for the https://github.com/psss/tmt/pull/132 (commit: https://github.com/psss/tmt/pull/132/commits/1d41c3f7d8028d9199b3064fd2ff01c22c48e30d)
```
2020-03-02 09:16:17 : CommitStatus.success : packit/testing-farm-fedora-30-x86_64
2020-03-02 09:15:37 : CommitStatus.success : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:13:39 : CommitStatus.success : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:10:17 : CommitStatus.failure : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:09:57 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:09:56 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:09:53 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:08:33 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:08:16 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:08:15 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:08:10 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:06:48 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-03-02 09:06:47 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-03-02 09:06:43 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-03-02 09:06:39 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:06:29 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:06:28 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:06:25 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:05:05 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:04:59 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-03-02 09:00:20 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 09:00:20 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 09:00:19 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 09:00:19 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-03-02 08:51:00 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-03-02 08:50:59 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-03-02 08:50:58 : CommitStatus.pending : packit/testing-farm-fedora-32-x86_64
2020-03-02 08:50:58 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:32:18 : CommitStatus.success : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:31:15 : CommitStatus.success : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:30:50 : CommitStatus.success : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:30:38 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:25:47 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:25:46 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:25:42 : CommitStatus.success : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:25:35 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:25:34 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:25:31 : CommitStatus.success : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:25:25 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:25:23 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:25:20 : CommitStatus.success : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:23:59 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:23:59 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:23:28 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:23:27 : CommitStatus.pending : packit/rpm-build-fedora-30-x86_64
2020-02-28 19:23:27 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:23:26 : CommitStatus.pending : packit/rpm-build-fedora-rawhide-x86_64
2020-02-28 19:23:26 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:23:25 : CommitStatus.pending : packit/rpm-build-fedora-31-x86_64
2020-02-28 19:13:34 : CommitStatus.success : continuous-integration/travis-ci/pr
2020-02-28 19:13:17 : CommitStatus.success : continuous-integration/travis-ci/push
2020-02-28 19:12:36 : CommitStatus.pending : continuous-integration/travis-ci/pr
2020-02-28 19:12:24 : CommitStatus.pending : packit/testing-farm-fedora-30-x86_64
2020-02-28 19:12:23 : CommitStatus.pending : packit/testing-farm-fedora-rawhide-x86_64
2020-02-28 19:12:23 : CommitStatus.pending : continuous-integration/travis-ci/pr
2020-02-28 19:12:23 : CommitStatus.pending : packit/testing-farm-fedora-31-x86_64
2020-02-28 19:12:22 : CommitStatus.pending : continuous-integration/travis-ci/pr
2020-02-28 19:12:20 : CommitStatus.pending : continuous-integration/travis-ci/push
2020-02-28 19:11:53 : CommitStatus.pending : continuous-integration/travis-ci/push
2020-02-28 19:11:52 : CommitStatus.pending : continuous-integration/travis-ci/push
```

**Comment by lachmanfrantisek** [comment]
Testing here: https://github.com/packit-service/hello-world/pull/77

**Comment by lachmanfrantisek** [comment]
Looks like it was fixed by my refactoring (#424 #426 ). Closing.

**Comment by psss** [comment]
Now looks as expected: https://github.com/psss/tmt/pull/133
Thanks for the fix!
